src/main/scala/com/amazon/deequ/analyzers/Analyzer.scala

```diff
@@ -303,17 +305,34 @@ object Preconditions {
     if (caseSensitive) {
       schema(column)
     } else {
-      schema.find(_.name.equalsIgnoreCase(column)).getOrElse {
+      schema.find(_.name.equalsIgnoreCase(removeEscapeColumn(column))).getOrElse {
         throw new IllegalArgumentException(s"Field {$column} doesn't not exist" )
       }
     }
   }
 
+  def removeEscapeColumn(column: String): String = {
+    if (column.endsWith("`")) {
+      column.substring(1, column.length - 1)
+    } else {
+      column
+    }
+  }
+
+  def escapeColumn(column: String): String = {
+    if (column.contains(".")) {
+      "`" + column + "`"
+    } else {
+      column
+    }
+  }
+
```

Comment:
Can we create a utility object which contains these methods? In the future, one wouldn't immediately know that the important methods for escaping/un-escaping are inside Analyzer.scala